action/protocol/staking/candidate_center.go

```diff
@@ -96,9 +138,10 @@ func (m *CandidateCenter) All() CandidateList {
 // Base returns the confirmed base state
 func (m CandidateCenter) Base() *CandidateCenter {
 	return &CandidateCenter{
-		base:   m.base,
-		size:   len(m.base.ownerMap),
-		change: newCandChange(),
+		base:     m.base,
+		size:     m.base.size(),
+		change:   newCandChange(),
+		hasAlias: m.hasAlias,
```

Comment:
how will this parameter be used in the returned candidate center? or you want to use it in `candBase.commit(change, hasAlias)`

Reply:
it is just to carry the same value of `hasAlias`
the `commit()` behavior depends on the `base` being `candBase struct {` or `candBaseFixAlias struct {`